src/Carousel.vue

```diff
@@ -273,9 +273,27 @@ export default {
     navigateTo: {
       immediate: true,
       handler(val) {
-        this.$nextTick(() => {
-          this.goToPage(val);
-        });
+        if (val.constructor === Array) {  
```

Comment:
rather than accessing the constructor here we can just compare `(typeof val === Array)`, it's a little easier to read since we're accessing `val[0]` and `val[1]` directly later in the method

Reply:
Will this do?

        // checking if val is an array, for an array typeof returns "object"
        if (typeof val === "object") {